kratos/utilities/time_discretization.h

```diff
@@ -0,0 +1,113 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Philipp Bucher
+//
+
+#if !defined(KRATOS_TIME_DISCRETIZATION_H_INCLUDED )
+#define  KRATOS_TIME_DISCRETIZATION_H_INCLUDED
+
+// System includes
+#include <vector>
+
+// External includes
+
+// Project includes
+
+namespace Kratos {
+namespace TimeDiscretization {
+
+class BDF1
+{
+public:
+    void ComputeBDFCoefficients(const double DeltaTime,
```

Comment:
As we discussed, but it's not critical.
```suggestion
    void ComputeBDFCoefficients(double DeltaTime,
```

Reply:
done